pkg/mimir/modules.go

```diff
@@ -304,6 +304,9 @@ func (t *Mimir) initServer() (services.Service, error) {
 	// Allow reporting HTTP 4xx codes in status_code label of request duration metrics
 	t.Cfg.Server.ReportHTTP4XXCodesInInstrumentationLabel = true
 
+	// Allow reporting gRPC statuses in status_code label of request duration metrics
+	t.Cfg.Server.ReportGRPCCodesInInstrumentationLabel = true
```

Comment:
Doing the change here causes the auto generated doc to not pick the new default (`docs/sources/mimir/configure/configuration-parameters/index.md` has not changed). To fix it you have to move it to `registerServerFlagsWithChangedDefaultValues()`.